src/styles/blocks/content.scss

```
@@ -0,0 +1,37 @@
+.content {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill,minmax(300px,1fr));
```

space between params
```suggestion
  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
```